debian/changelog

```diff
@@ -1,3 +1,13 @@
+cloud-init (23.3.1-0ubuntu2) mantic; urgency=medium
+
+  * Install missing dependencies in apt module if required for minimal images.
+    + d/p/cpick-0d9f149a-Pytestify-apt-config-test-modules-4424
+    + d/p/cpick-5023e9f9-Refactor-test_apt_source_v1.py-to-use-pytest-4427
+    + d/p/cpick-e9cdd7e3-Install-gnupg-if-gpg-not-found-4431
+    + d/p/cpick-015543d3-apt-install-software-properties-common-when-absent-but
```

Comment:
Good representing the individual quilt patches as cloud-init historically needs to represent separate debian directory packaging changes explicitly. 
We also need a way to call out that this is a "New upstream bug fix release based on 23.3.1" 
Are you able to include that as a scoping entry somehow?


```suggestion
  * New upstream bug fix release based on 23.3.1
    + Install missing dependencies in apt module if required for minimal images
      by the following debian/patches:
      - cpick-0d9f149a-Pytestify-apt-config-test-modules-4424
      - cpick-5023e9f9-Refactor-test_apt_source_v1.py-to-use-pytest-4427
      - cpick-e9cdd7e3-Install-gnupg-if-gpg-not-found-4431
      - cpick-015543d3-apt-install-software-properties-common-when-absent-but
```

Reply:
@blackboxsw Good point, done.